src/MultiTester/Summary.php

```diff
@@ -45,32 +43,46 @@ function_exists('posix_isatty') &&
     public function get()
     {
         $count = count($this->state);
-        $pad = max(array_map('strlen', array_keys($this->state)));
-        $successString = '%s    Success';
-        $failureString = '%s    > Failure!';
-        $successFinalString = '%d / %d     No project broken by current changes.';
-        $failureFinalString = '%d / %d     %s broken by current changes.';
-        $passed = 0;
-
-        if ($this->isColored()) {
-            $successString = "\033[42;97m $successString \033[0m";
-            $failureString = "\033[41;97m %s    Failure \033[0m";
-            $successFinalString = "\033[42;97m %d / %d     No project broken by current changes. \033[0m";
-            $failureFinalString = "\033[41;97m %d / %d     %s broken by current changes. \033[0m";
-        }
+        if ($count > 0) {
+            $pad = max(array_map('strlen', array_keys($this->state)));
+            $successString = '%s    Success';
+            $failureString = '%s    > Failure!';
+            $successFinalString = '%d / %d     No project broken by current changes.';
+            $failureFinalString = '%d / %d     %s broken by current changes.';
+            $passed = 0;
 
-        $output = '';
+            if ($this->isColored()) {
+                $successString = "\033[42;97m $successString \033[0m";
+                $failureString = "\033[41;97m %s    Failure \033[0m";
+                $successFinalString = "\033[42;97m %d / %d     No project broken by current changes. \033[0m";
+                $failureFinalString = "\033[41;97m %d / %d     %s broken by current changes. \033[0m";
+            }
 
-        foreach ($this->state as $package => $success) {
-            $passed += $success ? 1 : 0;
-            $output .= sprintf($success ? $successString : $failureString, str_pad($package, $pad)) . "\n";
-        }
+            $output = '';
 
-        $output .= "\n" . sprintf($passed === $count ? $successFinalString : $failureFinalString, $passed, $count, ($count - $passed) . ' project' . ($count - $passed > 1 ? 's' : '')) . "\n";
+            foreach ($this->state as $package => $success) {
+                $passed += $success ? 1 : 0;
+                $output .= sprintf($success ? $successString : $failureString, str_pad($package, $pad)) . "\n";
+            }
 
+            $output .= "\n" . sprintf($passed === $count ? $successFinalString : $failureFinalString, $passed, $count,
+                    ($count - $passed) . ' project' . ($count - $passed > 1 ? 's' : '')) . "\n";
+        } else {
+            $output = 'No project(s) tested, empty config file?' . PHP_EOL;
```

Comment:
This line is not covered by unit tests. Can you consider adding a unit test with an empty config that verifies you get the errors.

Typo: `No projects` = `0 projects` and in English, 0 is plural so you don't need the parentheses.

Please use `"\n"` instead of `PHP_EOL` for consistency. If for some reason we want to support other end-of-lines, it should be changed everywhere.

And maybe we can have a more helpful message: `check you have projects in your config file: ' . $this->getMultiTesterFile()`

Reply:
I've not yet figured out how to cover that line, so I've just ignored it for now